lib/util.js

```diff
@@ -366,9 +366,7 @@ const contractAddExtraAssetIcons = (file, imagesDstPath) => {
     symbol: "PDAI",
     decimals: 18
   }
-  fs.copyFileSync("resources/pdai.png", path.join(imagesDstPath, "pdai.png"))
   // Just copy ETH icon as ETH is not a contract token
-  fs.copyFileSync("resources/eth.png", path.join(imagesDstPath, "eth.png"))
```

Comment:
These will be included via https://github.com/brave/token-lists/tree/main/images